tools/make/lib/install/node.mk

```diff
@@ -21,27 +21,6 @@
 # Define the path to the root `package.json`:
 ROOT_PACKAGE_JSON ?= $(ROOT_DIR)/package.json
 
-
-# VARIABLES #
-
-#/
-# Installs package dependencies by executing [`npm install`][1].
-#
-# ## Notes
-#
-# -   Packages will be installed in a local `node_modules` directory relative to the project's `package.json` file.
-#
-# [1]: https://docs.npmjs.com/cli/install
-#
-# @example
-# make install-node
-#/
-install-node: $(ROOT_PACKAGE_JSON)
```

Comment:
I think you can still keep this command around; just remove the clone logic. While the dev guide can use `npm install`, having this command around for folks coming from the main project repo is still useful.